Controller/Index/index.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace SomethingDigital\AuctionItem\Controller\Index;
+
+use Composer\DependencyResolver\Request;
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\View\Result\PageFactory;
+/**
+ * Custom Controller for Auction Item display Page
+ *
+ * Class Index
+ * @package SomethingDigital\AuctionItem\Controller\Index
+ */
+class Index implements ActionInterface
+{
+    /**
+     * @var PageFactory
+     */
+    protected $pageFactory;
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
+     * @param PageFactory $pageFactory
+     * @param RequestInterface $request
+     */
+    public function __construct(
+        PageFactory $pageFactory,
+        RequestInterface $request
+    ) {
+        $this->pageFactory = $pageFactory;
+        $this->request = $request;
+    }
+
+    /**
+     * Execute function that returns page
```

Comment:
Personally I would omit comments like this one and the class-level comment. I the docblock standard says that comments should be included, but I feel like in situations like this where the function/class is self explanatory it adds extra noise. Totally okay if you disagree and want to keep them.